runtime/compiler/build/files/common.mk.ftl

```diff
@@ -241,6 +241,7 @@ JIT_PRODUCT_BACKEND_SOURCES+=\
     omr/compiler/optimizer/TranslateTable.cpp \
     omr/compiler/optimizer/TrivialDeadBlockRemover.cpp \
     omr/compiler/optimizer/UnionBitVectorAnalysis.cpp \
+    omr/compiler/optimizer/UnsafeSubexpressionRemoval.cpp \
```

Comment:
I think you meant `UnsafeSubexpressionRemover.cpp`; shouldn't a change like this belong in OMR?

Reply:
Thanks for catching that!  I guess I renamed it at some point, but I forgot to update this.

Yes, this ought to be in OMR, but my understanding is that OMR doesn't have any UMA build related things &mdash; at least not for the JIT.

Reply:
Corrected in commit 095a9d5691f3dd7ebcb4ac7675e29ac55f18f687.